### PR TITLE
support request body

### PR DESCRIPTION
**Pull request by shawncx**
1. support get/post/put/delete/patch
2. support request body
3. support api-version in query parameter

### CODE REVIEW

File: powershell/module/module-class.ts [hunk 1]
```diff
@@ -379,6 +379,21 @@ export class NewModuleClass extends Class {
 
   createInitAndPipeline(namespace: Namespace) {
     const $this = this;
+
+    const customEventListenerFunc = System.Func(
```
Comment (dolauli) [inline powershell/module/module-class.ts:383]: This file change looks from the master instead of modelerfour branch

Comment (shawncx) [reply powershell/module/module-class.ts]: Fixed. Thanks

**Pull request merged**